echo/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
let's add **server.port** , because we will have lots of app run at then same time.